docs/How-to-test-the-application.md

```diff
@@ -0,0 +1,59 @@
+# How to test App:
+- Using Physical device and apk
+- Using Emulator and Apk
+- Directly Running in Android Studio ( Recommended )
+
+
+## For option one or two:
+
+Every time a new commit is merged into the `develop` branch an _apk_ is released in the `apk` branch. To Navigate to `apk` Branch see here: 
+
+![Branch chnage](https://user-images.githubusercontent.com/16835617/92631953-8a52a900-f2ef-11ea-9e61-60692555283b.png)
+
+
+If you want to do manual testing on the application without setting up the development environment, you can download this [app-debug.apk](https://github.com/anitab-org/mentorship-android/blob/apk/app-debug.apk) and install it on your Android device.
+
+![apk-branch](https://user-images.githubusercontent.com/16835617/97575784-42105700-1a13-11eb-8a73-4669b8ebb343.png)
+
+
+![apkInstall](https://user-images.githubusercontent.com/16835617/97575669-1db47a80-1a13-11eb-8074-bcac49ee59db.png)
+
+
+For manually testing in physical device tick the option here :
+![Screenshot of Unknown install option ](https://static.apkpure.com/www/static/imgs/unknown_sources.jpg)
+
+Now click the apk to install !
+
+## For Testing on Emulator with APk :
+
+1. Now Open the project in Android Studio
+2. Wait untill all background process are finished 
+3. click the ADV button or click from already created devices in Android studio click 
+![advPress](https://user-images.githubusercontent.com/16835617/92633082-de5d8d80-f2ef-11ea-9593-ffb3646e8b55.png)
+4. For information on how to create an Emulator and Run it visit [Creating an ADV](https://developer.android.com/studio/run/managing-avds#createavd)
+5. Now Run the Emulator you have created 
+![RunADV](https://user-images.githubusercontent.com/16835617/92633141-f59c7b00-f2ef-11ea-9e3b-f686995fc57f.png)
+6. Navigate to the apk file downloaded 
+![apkAndEmulator](https://user-images.githubusercontent.com/16835617/92633190-0b11a500-f2f0-11ea-9c0b-2d4d7ee76162.png)
+7. Drag and Drop the Apk file into the Emulator.
+8. Now just look for the App in App tray and click to open.
+![gif](./images/gifDemo.gif)
+
+
+## For running through Android Studio
+1. [Clone the repository](https://github.com/anitab-org/mentorship-android/wiki/Fork,-Clone-&-Remote) follow the instructions here.
+2. Repeat to the step 5 of **Testing on Emulator with APk** 
```

Comment:
if you change it to `Repeat steps 1-5 of ...`, it would add a bit more clarity to this